src/gateway_gatt_char_wifi_ssid.erl

```diff
@@ -51,53 +52,96 @@ read_value(State=#state{value=Value}, _) ->
     {ok, Value, State}.
 
 
-handle_signal(SignalID, _Msg, State=#state{wifi_signal=SignalID}) ->
-    %% WiFi overall state has changed
-    {noreply, maybe_notify_value(update_value(State))};
+handle_signal(SignalID, Msg, State=#state{wifi_signal=SignalID}) ->
+    %% WiFi interface state has changed
+    Args = ebus_message:args(Msg),
+    lager:info("wifi_signal: ~p", [Args]),
+    case Args of
+        {ok, ["Connected", true]} ->
+            {noreply, maybe_notify_value(update_value(State, "online"))};
+        {ok, ["Connected", false]} ->
+            {noreply, maybe_notify_value(update_value(State, "disconnect"))};
+        _ ->
+            {noreply, State}
+    end;
 handle_signal(SignalID, Msg, State=#state{service_signal=SignalID}) ->
     %% Signal for the current service. Deal only with changes to the
     %% State attribute
-    case ebus_message:args(Msg) of
-        {ok, ["State", _]} ->
-            {noreply, maybe_notify_value(update_value(State))};
+    Args = ebus_message:args(Msg),
+    lager:info("service_signal: ~p", [Args]),
```

Comment:
This could be noisy

Reply:
and nothing really changed here code-wise?

Reply:
Yes, this log message is noisy. I'll remove it.